packages/ra-no-code/src/ui/useImportResourceFromCsv.tsx

```diff
@@ -24,23 +25,22 @@ export const useImportResourceFromCsv = (
         setParsing(true);
         parse<Record>(file, {
             header: true,
-            complete: async ({ data }) => {
+            skipEmptyLines: true,
+            complete: async ({ data, meta }) => {
                 const resourceAlreadyExists = !!resources[resource];
-
+                const records = sanitizeRecords(
+                    data.filter(record => !!record.id),
+                    meta
+                );
                 await Promise.all(
-                    data.map(record => {
-                        if (record.id) {
-                            return dataProvider
-                                .create(resource, {
-                                    data: record,
-                                })
-                                .catch(error => console.error(error));
```

Comment:
why did you remove error catching?

Reply:
Because I only used it for debug. A proper error catching should be added

Reply:
For now I'd rather have the whole thing explode than the error being hidden like before